community.dbuild

```diff
@@ -820,6 +820,8 @@ build += {
     ]
   }
 
+  // forked (December 2016) to get rid of the usual bintray-sbt stuff that
+  // makes dbuild upset
```

Comment:
@SethTisue Please check: https://github.com/typesafehub/dbuild/commit/7ef6be74b8025033d61bb14011e52a27c9b44723
Bintray-sbt based projects should be ok with 0.9.7-RC1. In theory, at least.